src/components/Project/AragonClient.js

```diff
@@ -113,14 +115,14 @@ class AragonClient extends React.Component {
               <Entry>
                 <Module>
                   <div className="badge-container">
-                    <StyledBadge>Agent application</StyledBadge>
+                    <StyledBadge>New app</StyledBadge>
                     <AragonOneBadge>Aragon One</AragonOneBadge>
                   </div>
-                  <h3>Agent application</h3>
+                  <h3>Aragon Agent</h3>
                   <h4>
-                    This application will enable organizations to interact
-                    natively with other web3 applications (including other
-                    Aragon organizations).
+                    This new Aragon app enables organizations to interact
+                    with other Ethereum protocols or dApps. Agent also allows organizations
```

Comment:
> with other Ethereum protocols or dApps 

I'm trying to get away from the "dapp" term, and "protocols" might be confusing since people usually think of the "Ethereum protocol" in the context of Ethereum and think of the "things that are deployed on-chain" as contracts.

So maybe we can simplify this and just say:

> with other Ethereum contracts.